src/components/Tune/TuneInput.vue

```diff
@@ -34,18 +35,23 @@ const props = withDefaults(
     maxLength: undefined,
     readonly: false,
     disabled: false,
-    definition: {}
+    definition: {},
+    alwaysShowError: false
   }
 );
 
 defineEmits(['update:modelValue']);
 
 const inputRef = ref();
 
-const showErrorMessage = ref(false);
+const forceError = ref(false);
+
+const showErrorMessage = computed(() => {
+  return forceError.value || props.alwaysShowError;
+});
```

Comment:
This is triggering a warning, as you're trying to edit this `showErrorMessage` value on line 99 and 100, but it's now a read-only computed value instead of `ref`